Kconfig

```diff
@@ -31,6 +31,12 @@ config GUESTAGENT_ARCH_RISCV64
 		Build lima-guestagent for "riscv64" Arch
 	default y
 
+config GUESTAGENT_ARCH_S390X
+	bool "guestagent Arch: s390x"
+	help
+		Build lima-guestagent for "s390x" Arch
+	default y
```

Comment:
you could have this off by default (`n`), if it is experimental

Reply:
We have been compiling other experimental archs (riscv64, armv7l) by default.

Let's revisit this in a separate issue/PR.

Reply:
- https://github.com/lima-vm/lima/issues/3321